networking/ptp/about-ptp.adoc

```diff
@@ -8,6 +8,11 @@ toc::[]
 
 Precision Time Protocol (PTP) is used to synchronize clocks in a network. When used in conjunction with hardware support, PTP is capable of sub-microsecond accuracy, and is more accurate than Network Time Protocol (NTP).
 
+[IMPORTANT]
+====
+If your cluster uses the OVN-Kubernetes network plugin and PTP uses the User Datagram Protocol (UDP) for hardware time stamping, the hardware time stamping cannot be applied to primary interface devices, such as an Open vSwitch (OVS) bridge.
+====
+
```

Comment:
How about add same note in sdn migration to OVN part?

Reply:
Hi @zhaozhanqi . I added the same note to the migration from SDN to OVNK doc on another PR that targets 4.16 and earlier versions (where SDN docs were not removed): https://github.com/openshift/openshift-docs/pull/92083/files

Would you be OK to approve both these PRs?

Reply:
thanks.  @dfitzmau this looks to me
for migration part, I suggest do a small update, thanks.